server.js

```diff
@@ -10,6 +10,11 @@ const queryString = require('query-string');
 const semver = require('semver');
 const xml2js = require('xml2js');
 const xpath = require('xpath');
+const Raven = require('raven');
+
+const serverSecrets = require('./lib/server-secrets');
+Raven.config(process.env.SENTRY_DSN || serverSecrets.sentry_dsn).install();
+Raven.disableConsoleAlerts();
```

Comment:
Without `disableConsoleAlerts` Raven will log all errors occurred while sending data to Sentry server https://github.com/getsentry/raven-node/blob/dcb7f7cdcceb0ff4042a104a5ef66f7e51716003/lib/client.js#L106.